proposals/collection-literals.md

```diff
@@ -207,57 +203,37 @@ Through the use of the [`init`](#init-methods) modifier, existing APIs can direc
 
     Here, if `b` is false, it is not required that any value actually be constructed for the empty literal since it would immediately be spread into zero values in the final literal.
 
-
 ## Natural type
 [natural-type]: #natural-type
 
 In the absence of a *constructible collection target type*, a non-empty literal can have a *natural type*.
 
 The *natural type* is determined using the [*best common type*](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#116315-finding-the-best-common-type-of-a-set-of-expressions) algorithm.
 
-A [*natural element type*](#natural-element-type) `T` is first determined.  If that cannot be determined, the literal has no *natural type*.  If `T` can be determined and it is some `KeyValuePair<TKey, TValue>`, then the *natural type* of the collection is `Dictionary<TKey, TValue>`; otherwise, the *natural type* of the collection is `List<T>`.
+A *natural element type* `T` is first determined from the *best common type* of:
+* The *type* of each element `e_n`, and
+* The *iteration type* of each element `..s_n`.
 
-If the *natural type* of the collection is `List<T>` the literal is not allowed to contain a `dictionary_element`.
+If a *natural element type* `T` cannot be determined, the literal has no *natural type*.  If `T` can be determined, the *natural type* of the collection is `List<T>`.
 
-This means there is no way for a literal to have a *natural type* of some `List<KeyValuePair<TKey, TValue>>` (though it certainly can be *target-typed* to that type).
+_Should `IEnumerable` contribute a natural element type of `object` or no contribution?_
+
+The choice of `List<T>` rather than `T[]` or `ImmutableArray<T>` is to allow mutation of `var` locals after initialization. `List<T>` is preferred over `Span<T>` because `Span<T>` cannot be used in `async` methods.
 
 ```c#
-var x = [];     // error: element type cannot be inferred for []
-var y = [1, 2]; // ok: List<int>
-object z = [3]; // ok: List<int>
+var list = [1, 2, 3];
+// ...
+list.Add(4);
 ```
 
-### Natural Element Type
-[natural-element-type]: #natural-element-type
+The compiler is free to use a representation other than the natural type if the difference is not observable.
```

Comment:
not sure what this means.  Is it that we could make our own subtype of List<T>?  Otherwise, i'm not sure what other representations for the natural type are possible that wouldn't be observable to the user.

Reply:
ignore this. :D